sdk/spring-sdk/src/it/java/com/example/wiring/actions/echo/ActionWithPrimitives.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.wiring.actions.echo;
+
+import kalix.javasdk.action.Action;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+
+public class ActionWithPrimitives extends Action {
+
+  @GetMapping("/action/{doubleValue}/{floatValue}/{intValue}/{longValue}")
```

Comment:
can we have a boolean in the path just for completeness?

Reply:
there is a `booleanValue` as a query param, should I add it also for path param?

Reply:
No, not needed. I see that `pathParamExtractors` also calls the new `toExtractor`. So we are covered.

I will merge it.